dao/src/main/java/com/carhouse/dao/impl/CarSaleDaoImpl.java

```diff
@@ -122,14 +134,24 @@ public CarSale getCarSale(final int carSaleId) {
      * Adds car sale advertisement to the end of list in database
      *
      * @param carSale the car sale
+     * @param file    the image file
      * @return car sale id
+     * @throws FileSystemException the file system exception when writing file
      */
     @Override
-    public Integer addCarSale(final CarSale carSale) {
+    public Integer addCarSale(final CarSale carSale, final MultipartFile file) throws FileSystemException {
         LOGGER.debug("method addCarSale with parameter: [{}]", carSale);
         KeyHolder keyHolder = new GeneratedKeyHolder();
+        if (!file.isEmpty()) {
```

Comment:
`?:` operator

Reply:
Ok, I will use it

Reply:
if (!file.isEmpty()) {
  carSale.setImageName(UUID.randomUUID().toString());
  fileWriter.writeFile(file, carSale.getImageName());
};
namedParameterJdbcTemplate.update(ADD_CAR_SALE_SQL, ...